.github/mergify.yml

```diff
@@ -8,6 +8,15 @@ pull_request_rules:
         branches:
           - humble
 
+  - name: Backport to jazzy at reviewers discretion
+    conditions:
+      - base=master
+      - "label=backport-jazzy"
+    actions:
+      backport:
+        branches:
+          - humble
```

Comment:
```suggestion
          - jazzy
```

Reply:
@christophfroehlich rest looks good to me

Reply:
oops, thanks!